test/onnx/test_fx_to_onnx.py

```diff
@@ -571,36 +510,6 @@ def test_fake_tensor_mode_huggingface_tiiuae_falcon(self):
             onnx.checker.check_model(export_output.model_proto)
             onnx.shape_inference.infer_shapes(export_output.model_proto)
 
-    @pytorch_test_common.skip_in_ci(
-        "Not decorated with xfail because CI doesn't have enough memory to run and then fail."
-        "torch._dynamo.exc.UserError: Dynamic control flow is not supported at the moment. "
-        "Please use functorch.experimental.control_flow.cond to explicitly capture the control flow"
-    )
-    def test_fake_tensor_mode_huggingface_mosaicml_mpt_7b(self):
```

Comment:
was it renamed to `test_fake_tensor_mode_huggingface_mosaicml_mpt` on purpose? any arch change?

Reply:
Yes the name 7b is not true anymore, as it's built from config.